maquetado/scss/details.scss

```diff
@@ -1,14 +1,37 @@
 .book,
 .book-element,
-.book-elements {
+.book-elements,
+.book-title {
   display: flex;
 }
 
+.book {
+  margin: 10px;
```

Comment:
Same, avoid using margin top or left. If you want to separate this from your browser border I would add padding inside its parent instead.